### PR TITLE
test: update helm chart version to the latest

### PR DESCRIPTION
**Pull request by randmonkey**
**What this PR does / why we need it**:

Since in charts 2.26.1 we fixed the parsing of semvers, we update the charts to the latest 2.26.3. 

**Which issue this PR fixes**:

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: test/consts/helm.go [hunk 1]
```diff
@@ -1,6 +1,4 @@
 package consts
 
 // KongHelmChartVersion is the version of the Kong Helm chart to use in tests.
-// TODO: use chart 2.25.0 as a workaround before charts deal with invalid semver in image tag properly
-// https://github.com/Kong/charts/issues/856
-const KongHelmChartVersion = "2.25.0"
+const KongHelmChartVersion = "2.26.1"
```
Comment (pmalek) [inline test/consts/helm.go:4]:
```suggestion
const KongHelmChartVersion = "2.26.2"
```

https://github.com/Kong/charts/releases/tag/kong-2.26.2

Comment (randmonkey) [reply test/consts/helm.go]: done.

**Review by pmalek — COMMENTED** [review]
The places that now mention e.g. 3.4 as effective version are susceptible to get outdated when we forget to update it. I believe we should at the very least update the release instruction so that we update those as well during the release. WDYT?

re: https://github.com/Kong/kubernetes-ingress-controller/blob/16bf8d7968aa6fe19b15cf913751fd1ad7076f7d/.github/ISSUE_TEMPLATE/release.yaml#L26-L34

**Comment by randmonkey** [comment]
> The places that now mention e.g. 3.4 as effective version are susceptible to get outdated when we forget to update it. I believe we should at the very least update the release instruction so that we update those as well during the release. WDYT?
> 
> re:
> 
> https://github.com/Kong/kubernetes-ingress-controller/blob/16bf8d7968aa6fe19b15cf913751fd1ad7076f7d/.github/ISSUE_TEMPLATE/release.yaml#L26-L34

Yes, I think so. I am thinking of how to update it automatically when Kong releases.

**Comment by randmonkey** [comment]
> The places that now mention e.g. 3.4 as effective version are susceptible to get outdated when we forget to update it. I believe we should at the very least update the release instruction so that we update those as well during the release. WDYT?
> 
> re:
> 
> https://github.com/Kong/kubernetes-ingress-controller/blob/16bf8d7968aa6fe19b15cf913751fd1ad7076f7d/.github/ISSUE_TEMPLATE/release.yaml#L26-L34

Updated release issue template.

**Review by pmalek — APPROVED** [review]

**Pull request merged**